libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -71,7 +71,8 @@ proc init*(_: type[GossipSubParams]): GossipSubParams =
       ipColocationFactorThreshold: 1.0,
       behaviourPenaltyWeight: -1.0,
       behaviourPenaltyDecay: 0.999,
-      disconnectBadPeers: false
+      disconnectBadPeers: false,
+      disablePX: false
```

Comment:
peer exchange is a privacy leak in general, should be opt-in .. accordingly, better use `enablePX`